engine/src/physics/solver/integration/physics_constants_integration.hpp

```diff
@@ -0,0 +1,24 @@
+/// @file
+/// @brief Resource @ref cubos::engine::PhysicsConstantsIntegration.
+/// @ingroup physics-plugin
+
+#pragma once
+
+#include <cubos/core/reflection/reflect.hpp>
+
+#include <cubos/engine/api.hpp>
+
+namespace cubos::engine
+{
+    /// @brief Resource which allows configuration over constants in the integration plugin.
+    /// @ingroup physics-plugin
+    struct PhysicsConstantsIntegration
+    {
+        CUBOS_REFLECT;
+
+        float minInvMass = 0.0F;
+
+        float minInvInertia = 0.0F;
```

Comment:
Ok, one last thing that I didn't notice last time, the minInvInertia should be the matrix itself. This avoids creating a new matrix every time we do a comparison with it.
```suggestion
        float minInvInertia = glm::mat3(0.0F);
```